content/blog/2018/aws-nlb/index.md

```diff
@@ -16,8 +16,11 @@ Network load balancer (NLB) could be used instead of classical load balancer. Yo
 
 The following instructions require a Kubernetes **1.9.0 or newer** cluster.
 
-{{< warning_icon >}} Usage of AWS `nlb` on Kubernetes is an Alpha feature and not recommended for production clusters.
-{{< warning_icon >}} Usage of AWS `nlb` does not support the creation of two or more Kubernetes clusters running Istio in the same zone as a result of [Kubernetes Bug #69264](https://github.com/kubernetes/kubernetes/issues/69264).
+{{< warning >}}
+Usage of AWS `nlb` on Kubernetes is an Alpha feature and not recommended for production clusters.
+
```

Comment:
this part was updated in the blog post (by me sadly).  Should we add the tip date?

Reply:
Done, I added a last_update front-matter entry and a tip section about the blog having been updated.